Makefile

```diff
@@ -27,9 +27,11 @@ proto:
 	done
 
 test: testdeps
+	go test -i google.golang.org/grpc/...
```

Comment:
Yikes, that's really odd. It means that you can't use Go's incremental compilation for your tests. OK, I backed out this commit.